addons/delivery/delivery.py

```diff
@@ -228,6 +253,24 @@ def get_price(self, cr, uid, id, order, dt, context=None):
         total = self.pool['res.currency'].compute(cr, uid, order.currency_id.id, order.company_id.currency_id.id, total, context=ctx)
         return self.get_price_from_picking(cr, uid, id, total,weight, volume, quantity, context=context)
 
+    def get_cost_from_picking(self, cr, uid, id, total, weight, volume, quantity, context=None):
```

Comment:
As I did in other PRs to Odoo, I prefer to follow Odoo's nomenclature and styles. In order to respect their ways. I will wait for their comments before change that name.
BTW, @pedrobaeza thanks for your review and comments